packages/dx-react-chart-bootstrap4/src/plugins/title.jsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { Title as TitleBase } from '@devexpress/dx-react-chart';
+import { Text } from '../templates/title/text';
+
+export class Title extends React.PureComponent {
+  render() {
+    return (
+      <TitleBase
+        titleComponent={Text}
```

Comment:
Maybe rename this property to `textComponent` or `rootComponent` ?

Reply:
textComponent, nice